lib/keywords.js

```diff
@@ -81,3 +81,8 @@ export const loadDialect = (dialectId, dialect, allowUnknownKeywords = false) =>
       }
     });
 };
+
+export const loadSchemaDialects = () => {
```

Comment:
The name of this function doesn't make sense. It's not loading dialects.

Reply:
what about getDialects?

Reply:
That's better, but how about "getDialectIds"? It's not the dialect that it's returning, but the identifiers for the dialects.

Reply:
Yeah sounds perfect.